lib/Config/Migrator.php

```diff
@@ -448,6 +449,26 @@ function statisticsForms() {
     return $this->sqlify(__FUNCTION__, $attributes);
   }
 
+  function statisticsWoocommerceOrders() {
+    $attributes = array(
+      'id int(11) unsigned NOT NULL AUTO_INCREMENT,',
+      'newsletter_id int(11) unsigned NOT NULL,',
+      'subscriber_id int(11) unsigned NOT NULL,',
+      'queue_id int(11) unsigned NOT NULL,',
+      'click_id int(11) unsigned NOT NULL,',
+      'order_id bigint(20) unsigned NOT NULL,',
+      'order_currency varchar(10) NOT NULL,',
```

Comment:
why this isn't `char(3)`, all currencies should have a code, don't they?

Reply:
Yeah, maybe... but this is WP/WC world so what do I know? They use meta tables so they are not limited to 3 chars. What if someone uses unusual codes due to some political/inflation/deflation changes in a country or simply because the want? Like `USD-old`, `USD-new` or something?

Reply:
Turns out there is even standard for that. All currencies have 3 characters code: https://en.wikipedia.org/wiki/ISO_4217

I was thinking that having `char` instead of `varchar` creates a table with a static row length and that could be better for performance.

But if you think it is not worth it because in the WP world nothing is stable I will not argue with you.

Reply:
I browsed through WC code and it seems you're right, they use 3-char codes everywhere.